app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,45 @@
+<resources>
+    <string name="app_name">Medical Helper</string>
+    <string name="email">email</string>
+    <string name="UserId">User Id</string>
+    <string name="Password">Password</string>
+    <string name="ConfirmPassword">Confirm Password</string>
+    <string name="error_message_email">Enter Valid Email</string>
```

Comment:
Please be consistent in naming of variables. ConfirmPassword is camelcased and error_message_email is dash separated. Camel casing is widely popular, I recommend you to use it. 
refer: https://en.wikipedia.org/wiki/Camel_case